src/lib/error-handling/index.ts

```diff
@@ -51,7 +52,10 @@ export class PizzlyError extends Error {
         this.status = 404
         this.message = 'That authentication ID could not be found on the database.'
         break
-
+      case 'token_refresh_failed':
```

Comment:
https://github.com/Bearer/Pizzly/pull/43/files#diff-0ba7ceaa154c56ca7d5c57ccb7aa4683R16

We should create different error types and handle statuses somewhere else so that this clas does not contain too much responsibilities.